docs/cli-client/htlc/README.md

```diff
@@ -0,0 +1,35 @@
+# iriscli htlc
+
+## Description
+
+Htlc allows you to manage local htlc for atomic swap with other chains
+
+## Usage
+
+```bash
+ iriscli htlc [command]
+```
+
+## Available Commands
+
+| Name                        | Description                 |
+| --------------------------- | --------------------------- |
+| [create](create.md)         | Create a HTLC               |
```

Comment:
a HTLC --> an HTLC
a htlc --> an HTLC

Please make the changes throughout the document.